fix(deps): move demo deps to dev deps

waterplea opened the pull request:
Those deps that you used for demo page are not actually dependencies of the vocoder itself but are used for demo only. So they should go into devDeps in case you would ever want to publish it to `npm`. BTW, do you plan doing so? :) I could use it my app :P

olvb commented:
Hi, thanks for the PR! I suppose you are right, these deps are indeed dev dependencies only needed for the demo. But then so are the other deps (`http-server` and the `wave-*`stuff), the core of this project being only the Audio Worklet itself.
As regards the npm package, I don't know enough about the js ecosystem to maintain a package myself but if you feel like it go for it. I can licence the repo under a permisive licence if that is needed.

waterplea commented:
Yes, those other deps are already in devDep :) The only dependency is `fft.js` which I left in `dependencies`. I've updated package.json to include terser and a minification step, added command to publish that builds and minifies the vocoder then pushes it to npm (using added files field). I also renamed the project to `phaze-vocoder` as `phaze` is already taken on npm. Looks like it should be ready for publishing now.

olvb commented:
Aaaah yes indeed, got completely mixed up! Ok then.
